torchaudio/models/wav2vec2/model.py

```diff
@@ -27,10 +27,10 @@ class Wav2Vec2Model(Module):
             Auxiliary module. If provided, the output from encoder is passed to this module.
     """  # noqa: E501
     def __init__(
-            self,
```

Comment:
Could you revert the unrelated changes?

Reply:
Sure, will do. Sorry I didn't notice the indentations when I was reviewing the PR. Do you think we should fix them in a separate PR?